linkml/generators/pydanticgen/template.py

```diff
@@ -212,6 +212,10 @@ class PydanticAttribute(TemplateModel):
     equals_number: Optional[Union[int, float]] = None
     minimum_value: Optional[Union[int, float]] = None
     maximum_value: Optional[Union[int, float]] = None
+    exact_cardinality: Optional[int] = None
+    minimum_cardinality: Optional[int] = None
+    maximum_cardinality: Optional[int] = None
+    multivalued: Optional[bool] = None
```

Comment:
super subtle thing - since the fields in template models are considered to be "the things that the template is capable of representing" it's good to have multivalued in here (and i see why you did it in the template too), could we remove it from `meta_exclude` above which is "the things that are represented in other ways than explicitly as a field in the template model and should be excluded when embedding schema metadata in the generated pydantic field"

Reply:
Do we want to remove `range` as well then?

Reply:
yes, but no need to do it in this PR. i am planning on doing some updates here in how this is handled soon. for now it's just a redundant value which isn't the worst thing in the world.